ackama_rails_template.config.yml

```diff
@@ -23,6 +23,9 @@ apply_variant_bootstrap: false
 # Generates code to setup a fairly generic Capistrano install
 apply_variant_deploy_with_capistrano: false
 
+# Copy in some examples for Github Actions CI
+apply_variant_github_actions_ci: false
```

Comment:
Is there any reason why this shouldn't be enabled by default? I like keeping it as its own variant that can be disabled, but we're investing more and more in having GitHub Actions be our One True CI system, so I think its pretty safe to have as a default.